### ISSUE TITLE
How long for training the model?

### ISSUE THREAD

**Issue by ShenZheng2000** [hhb072/Moss#2]
Hello, authors. I really appreciate your nice work on image draining. I would like to know how long it took to converge your model. Are you using a single GPU for both training and testing?
Thanks in advance.

**Comment by BinLi-git** [comment]
> Hello, authors. I really appreciate your nice work on image draining. I would like to know how long it took to converge your model. Are you using a single GPU for both training and testing? Thanks in advance.

do you get train code?

**Comment by ShenZheng2000** [comment]
No. It has been more than one year since I opened this issue, but the authors still have not shared the training code.
